docs/8-call-status.md

```diff
@@ -1,75 +1,34 @@
-# Nexmo In-App Call States 
+# Call States 
```

Comment:
Statuses

Reply:
same for the entire document.
vapi uses 'statuses'
so lets be consistent
it was my bad using the word 'states' loosly

Reply:
κατάσταση